heynovaio/designstudio
# Footer palm/template wrangling

Task: https://app.clickup.com/t/9009201449/BUILD-382

Just a little guy to add the palm illustration and sort out how we wanted to do it. Landed on good ol' svg mask so we can change the colour easily. Now that this is in the footer we can steal the code for other components on the site. 

To test: 
- Pull branch & yarn install
- yarn start
- Observe the palm illustration in the footer and make sure it's looking fly. 
- I'm a little worried about the contrast, I actually boosted it from the designs so thoughts?
